pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -434,6 +434,16 @@ func (m *kubeGenericRuntimeManager) computePodContainerChanges(pod *v1.Pod, podS
 		ContainersToKill:     make(map[kubecontainer.ContainerID]containerToKillInfo),
 	}
 
+	// Get a ref for Event logs. If this fails, podRef will be nil, but the Eventf handler tolerates this.
+	podRef, err := ref.GetReference(api.Scheme, pod)
+	if err != nil {
+		glog.Errorf("Couldn't make a ref to pod %q: '%v'", format.Pod(pod), err)
+	}
+
```

Comment:
See comment.  Eventf handles ref being nil, and it's not a reason to stop going.  (This happens in several other places in the kubelet code, I'm just following convention)